doc/integrator/ngeo.rst

```diff
@@ -108,3 +108,23 @@ The map configuration will be here:
    If you previously had a CGXP application and want to keep your permalinks
    compatible, the resolutions should be the same as in the previous
    application.
+
+Dynamic.js view
+---------------
+
+To configure the ngeo constance with dynamic or configurable bales we have the dynamic view.
```

Comment:
constance => constants

Reply:
bales ?

Reply:
I don't understand you second message...

Reply:
I just understand, fixed, thanks :-)